packages/commonwealth/client/styles/shared.scss

```diff
@@ -21,17 +21,11 @@ $sidebar-community-header-height: 56px;
 
 // base fonts
 
-$fonts: NeueHaasUnica, Silka, 'Segoe UI', sans-serif, 'Apple Color Emoji',
+$fonts: Silka, NeueHaasUnica, 'Segoe UI', sans-serif, 'Apple Color Emoji',
```

Comment:
This for sure is not something we want. You are changing default font from `NeueHaasUnica` to `Silka` in entire App which is definitely not expected